pkg/service/service.go

```diff
@@ -17,6 +17,8 @@ package service
 import (
```

Comment:
All changes in this file are not necessary. Please read my review comment.

Reply:
@aanm so do you say that we don't need to touch pkg/service/service.go at all. The reason we added code in here is to capture the UpdatedTime when the service is updated.